meeting-notes/tbdex/2024-06-10.md

```diff
@@ -0,0 +1,8 @@
+# 2024-06-10
```

Comment:
suggestion: let's create a new top-level directory for `working-groups` 

then within that we can have `tbdex`, `web5`, `rust`, etc and within those a directory for `meeting-notes`